distributed/protocol/cupy.py

```diff
@@ -7,36 +7,18 @@
 
 @cuda_serialize.register(cupy.ndarray)
 def serialize_cupy_ndarray(x):
-    # TODO: handle non-contiguous
-    # TODO: Handle order='K' ravel
-    # TODO: 0d
+    # Making sure `x` is behaving
+    if not x.flags.c_contiguous or True:
```

Comment:
Same question as in https://github.com/dask/distributed/pull/3047/files#r323343373